draft-ietf-rtcweb-jsep.xml

```diff
@@ -2146,12 +2146,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             lines are not meaningful in the JSEP context and
             MAY be discarded once they have been checked.
             <list>
+              <t>The "c=" line MUST be checked for syntax but its value SHOULD be ignored.
```

Comment:
Fine with me. @juberti, if you agree, say so and I will make the change and
merge.

On Mon, Jun 15, 2015 at 6:24 AM, Cullen Jennings <notifications@github.com>
wrote:

> In draft-ietf-rtcweb-jsep.xml
> <https://github.com/rtcweb-wg/jsep/pull/142#discussion_r32418537>:
>
> > @@ -2146,12 +2146,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
> >              lines are not meaningful in the JSEP context and
> >              MAY be discarded once they have been checked.
> >              <list>
> > +              <t>The "c=" line MUST be checked for syntax but its value SHOULD be ignored.
>
> suggestions ... change "SHOULD be ignored" to "is not used"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtcweb-wg/jsep/pull/142/files#r32418537>.
>